crates/holochain_p2p/src/test.rs

```diff
@@ -392,7 +392,7 @@ mod tests {
 
         p2p.join(dna.clone(), a1.clone()).await.unwrap();
         p2p.join(dna.clone(), a2.clone()).await.unwrap();
-        p2p.join(dna.clone(), a3.clone()).await.unwrap();
+        //p2p.join(dna.clone(), a3.clone()).await.unwrap();
```

Comment:
Is this broken?

Reply:
```suggestion
```

nah, it was just easier to modify the test so it didn't include this, due to the default node count values - I'll delete that commented line.